testing/tests/DevExpress.ui.widgets.editors/validationGroup.tests.js

```diff
@@ -57,6 +57,22 @@ QUnit.module('General', {
         assert.ok(validator.validate.calledOnce, 'Validator should be validated as part of group');
     });
 
+    QUnit.test('should find group if validator is child of validationGroup ', function(assert) {
+        const $group_container = $('#dxValidationGroup');
+        const group = this.fixture.createGroup($group_container);
+        const $validator = $group_container.dxValidator({
+            adapter: sinon.createStubInstance(DefaultAdapter)
+        });
+        const validator = $validator.dxValidator('instance');
+        validator.validate = sinon.spy(validator.validate);
+        $validator.appendTo($group_container);
+        triggerShownEvent($group_container);
+        ValidationEngine.validateGroup(group);
```

Comment:
Let's add an empty row before this to separate test "action" part from "arrange" part. [AAA](https://automationpanda.com/2020/07/07/arrange-act-assert-a-pattern-for-writing-good-tests/)

Reply:
done